test/acceptance/features/steps/steps.py

```diff
@@ -278,6 +278,40 @@ def then_envFrom_contains_intermediate_secret_name(context, intermediate_secret_
         intermediate_secret_name, wait=True, timeout=120) is not None, f"There is no deployment with intermediate secret {intermediate_secret_name}"
 
 
+# STEP
+@given(u'OLM Operator Backend is running')
+def backend_operator_manifest_installed(context):
```

Comment:
This is still not addressed: https://github.com/redhat-developer/service-binding-operator/pull/597#discussion_r481111836

Reply:
my comment here refers something else.

Reply:
Done https://github.com/redhat-developer/service-binding-operator/pull/597/commits/50de2de5c27875b806e76519efce63eb9f1be938